pkg/controller/controlplane/valuesprovider.go

```diff
@@ -127,6 +127,11 @@ var controlPlaneShootChart = &chart.Chart{
 				{Type: &rbacv1.ClusterRoleBinding{}, Name: "system:controller:cloud-node-controller"},
 			},
 		},
+		{
+			Name:    "metallb",
+			Images:  []string{packet.MetalLBControllerImageName, packet.MetalLBSpeakerImageName},
+			Objects: []*chart.Object{},
```

Comment:
I wasn't sure whether this field should be populated and with what values exactly. Things seem to behave correctly without this AFAICT. Please tell me if a change is necessary here :pray:

Reply:
I guess it's okay to not specify the objects because all the resources that the extension deploys into the shoot cluster are managed by the GRM.
This `Objects` was needed in an ancient time where GRM didn't exist and the extensions deployed (and deleted!) their resources themselves (by directly talking to the kube-apiserver).

Reply:
Got it. Will leave it as is then. Thanks for the explanation!